src/tokens/layout/bb-productive.json

```diff
@@ -1765,5 +1783,1651 @@
+              "$value": "{bb.size.fluid.50}"
+            }
+          }
+        }
+      }
+    },
+    "back_to_top": {
+      "space": {
+        "inset": {
+          "top": {
+            "$type": "other",
+            "$value": "{bb.size.fluid.50}"
+          },
+          "right": {
+            "$type": "other",
+            "$value": "{bb.size.fluid.150}"
+          },
+          "bottom": {
+            "$type": "other",
+            "$value": "{bb.size.fluid.50}"
+          },
+          "left": {
+            "$type": "other",
+            "$value": "{bb.size.fluid.150}"
+          }
+        }
+      }
+    },
+    "box": {
+      "heading": {
+        "space": {
+          "inset": {
+            "top": {
+              "$type": "other",
+              "$value": "{bb.size.fluid.150}"
+            },
+            "right": {
+              "$type": "other",
+              "$value": "{bb.size.fluid.150}"
+            },
+            "bottom": {
+              "$type": "other",
+              "$value": "{bb.size.0}"
+            },
+            "left": {
+              "$type": "other",
+              "$value": "{bb.size.fluid.150}"
+            }
+          }
+        }
+      },
+      "content": {
+        "space": {
+          "inset": {
+            "top": {
+              "$type": "other",
+              "$value": "{bb.size.fluid.150}"
+            },
+            "right": {
+              "$type": "other",
+              "$value": "{bb.size.fluid.150}"
+            },
+            "bottom": {
+              "$type": "other",
+              "$value": "{bb.size.fluid.150}"
+            },
+            "left": {
+              "$type": "other",
+              "$value": "{bb.size.fluid.150}"
+            }
+          }
+        }
+      }
+    },
+    "modal": {
+      "content": {
+        "space": {
+          "inset": {
+            "top": {
+              "$type": "other",
+              "$value": "{bb.size.fluid.50}"
+            },
+            "right": {
+              "$type": "other",
+              "$value": "{bb.size.fluid.150}"
+            },
+            "bottom": {
+              "$type": "other",
+              "$value": "{bb.size.fluid.150}"
+            },
+            "left": {
+              "$type": "other",
+              "$value": "{bb.size.fluid.150}"
+            }
+          }
+        }
+      },
+      "footer": {
+        "space": {
+          "inset": {
+            "top": {
+              "$type": "other",
+              "$value": "{bb.size.fluid.100}"
+            },
+            "right": {
+              "$type": "other",
+              "$value": "{bb.size.fluid.150}"
+            },
+            "bottom": {
+              "$type": "other",
+              "$value": "{bb.size.fluid.100}"
+            },
+            "left": {
+              "$type": "other",
+              "$value": "{bb.size.fluid.150}"
+            }
+          }
+        }
+      },
+      "header": {
+        "size": {
```

Comment:
Is `size` right here - or should that be `space` like the others? Figma spec seems to say `space`

Reply:
That sure looks like a mind lapse typo, nice catch.